### PR TITLE
beerchat.sound_play

### PR DESCRIPTION
**Pull request by S-S-X**
Add `beerchat.sound_play(name, sound)`

* Overall cleaner.
* Allows adding fairly easily per player config later.
* Wants #94 to be merged or direct rebase to master + possible conflict resolution.

### CODE REVIEW

File: common.lua [hunk 1]
```diff
@@ -73,6 +71,10 @@ beerchat.is_player_subscribed_to_channel = function(name, channel)
 		and (nil ~= beerchat.playersChannels[name][channel])
 end
 
+beerchat.sound_play = beerchat.enable_sounds and function (target, sound)
```
Comment (BuckarooBanzay) [inline common.lua:74]: smells like premature optimization here :smirk:

**Review by BuckarooBanzay — APPROVED** [review]

File: common.lua (hunk 1 above)
Comment (S-S-X) [reply common.lua:74]: It is more about moving check away from call site, intended to actually go inside that function later when actual per player and/or dynamic configuration is added. Currently it is hardcoded value in init.lua.

If you think syntax is bad idea I could change it right away and just leave check inside function? If you don't care then I'll leave it like it is for now and update when there's actually some way to configure it.

**Comment by wsor4035** [comment]
@S-S-X needs conflict resolving

**Comment by S-S-X** [comment]
Resolved, rebased and changed target branch to master.

**Review by wsor4035 — APPROVED** [review]
looks good, didnt test

**Pull request merged**